frontend/src/container/ReduxTlmDispatcher.jsx

```diff
@@ -74,7 +83,17 @@ export class ReduxTlmDispatcher extends React.Component {
   }
 
   handleContentCreated = data => {
-    this.props.dispatch(addWorkspaceContentList([data.content], data.workspace.workspace_id))
+    if (data.content.content_namespace === 'upload') {
+      this.props.dispatch(addWorkspaceShareFolderContentList(
+        [{
+          ...data.content,
+          parent_id: data.content.content_type === CONTENT_TYPE.FOLDER ? SHARE_FOLDER_ID : data.content.parent_id
```

Comment:
I'm confused here
shouldn't the folder have the proper parent_id already ?
I think the constant SHARE_FOLDER_ID's value should be the same as the backend one (which is -1 I think).
Is this test here only for safety ?

Reply:
If the folder is a sharedFolder the backend set the `parent_id` to `null` (API and TLM), I just followed this current implementation, [see](https://github.com/tracim/tracim/blob/6483195859bdbf76bee7092a937a186b592159a5/frontend/src/container/WorkspaceContent.jsx#L603)

Reply:
indeed.
It's correct then
you can resolve